config/rbac/role.yaml

```diff
@@ -152,31 +153,7 @@ rules:
   - opentelemetry.io
   resources:
   - opampbridges/status
-  verbs:
-  - get
-  - patch
-  - update
-- apiGroups:
-  - opentelemetry.io
-  resources:
-  - opentelemetrycollectors
-  verbs:
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - opentelemetry.io
-  resources:
   - opentelemetrycollectors/finalizers
-  verbs:
-  - get
-  - patch
-  - update
-- apiGroups:
-  - opentelemetry.io
-  resources:
```

Comment:
Why are those not needed anymore?

Reply:
This section was merged with the one for Instrumentations. I guess the new version of controller-tools is better at arranging RBAC rules.